.gitignore

```diff
@@ -125,3 +125,4 @@ buildSrc/src/main/kotlin/appPackage/BuildKonfig.kt
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+.secrets
```

Comment:
Why we need this?

Reply:
I kept some secrets to test the workflow locally, so I add the `.secrets` file to the gitignore